_posts/2018-09-21-ng-integration-tests.md

```diff
@@ -286,4 +286,6 @@ To ignore this message site-wide, create a file with the name and path `/data/wo
 
 The field *message* is compulsory and it must either be a string to be whitelisted, or a regular expression. If regular expressions are used, then there the field *regex* must also be set to *true*. The field *level* is not compulsory.
 
+Please note, because JSON uses the backslash character as an escape sign, backslashes used in regular expressions should be escaped. This means, the regular expression `jquery\.com` must be typed as `jquery\\.com`, for example.
```

Comment:
I would recommend using "escape character" instead of "escape sign".

Reply:
That sounds better! Changed it!